aries-site/src/pages/components/tabs.js

```diff
@@ -17,19 +23,132 @@ const Tabs = () => (
     />
     <ContentSection>
       <Subsection name={title} level={1} topic={topic}>
+        <SubsectionText>{page.description}</SubsectionText>
+        <Example
+          code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/tabs/TabsExample.js"
+          designer="https://designer.grommet.io/tabs?id=HPE-design-system-hpedesignsystem-hpe-com"
+          docs="https://v2.grommet.io/tabs?theme=hpe#props"
+          figma="https://www.figma.com/file/Kp4dWyhUTnKIJ1Cg5CoL9o/HPE-Tabs-Component"
+        >
+          <TabsExample />
+        </Example>
+      </Subsection>
+    </ContentSection>
+    <ContentSection>
+      <Subsection name="Guidance">
         <SubsectionText>
           Tabs allow a user to access content while maintaining the existing
-          context. It consists of a container, or box, with tab controls to
-          expose the contents of the container.
+          context. The Tabs component serves as the outer wrapper and contains
```

Comment:
They can be used to rapidly navigate between different content while maintaining existing context

Adding it navigating between tabs may be helpful

Reply:
Done. : )